var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -101,9 +105,20 @@ class Hiop(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("umpire", when="+raja")
     depends_on("raja+openmp", when="+raja~cuda~rocm")
 
-    # Umpire > 0.14 and RAJA > 6.0 require c++ std 14
-    depends_on("raja@0.14.0:0.14", when="@0.5.0:0.6.2+raja")
-    depends_on("umpire@6.0.0:", when="@0.5.0:0.6.2+raja")
+    # RAJA > 0.14 and Umpire > 6.0 require c++ std 14
+    # We are working on supporting newer Umpire/RAJA versions
+    depends_on("raja@0.14.0:0.14", when="@0.5.0:+raja")
+    depends_on("umpire@6.0.0:6", when="@0.5.0:+raja")
+    depends_on("camp@0.2.3:0.2", when="@0.5.0:+raja")
+
+    # This is no longer a requirement in RAJA > 0.14
+    depends_on("umpire+cuda~shared", when="+raja+cuda")
```

Comment:
You can be more specific here about the version of raja in the `when` argument.

Reply:
I fixed this, as well as another shared requirement in HiOp